internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/OfflinePushStatus.java

```diff
@@ -208,6 +208,10 @@ public Map<Integer, Map<CharSequence, Integer>> getIncrementalPushStatus(
     for (PartitionStatus partitionStatus: getPartitionStatuses()) {
       Map<CharSequence, Integer> partitionPushStatus = new HashMap<>();
       Partition partition = partitionAssignment.getPartition(partitionStatus.getPartitionId());
+      if (partition == null) {
```

Comment:
when this could happen?

Reply:
It happened a few times

Reply:
Do you happen to have the error message for this issue, I am wondering if InLogs is still available to reveal this. But I guess the guard here is still valid.

Reply:
And also, if we handle NPE by throw Exception here, then it is still throw Exception in controller? I am wondering at which level do we catch the exception and do some proper logging... Otherwise it does not change much, we just wrap the NPE with another exception (not sure if it is what you want though)

Reply:
Right. This change is not trying to resolve the true cause behind it but attempts to reveal more information otherwise it's very difficult to debug unless we have the heap dump when the incident happens.

Reply:
was wondering if there is some transient issue to update the routing repo. we could put NOT_CREATED status for that null partition in the map and log error ? ie    
   incPushStatusMap.put(partitionId, NOT_CREATED);

Reply:
> was wondering if there is some transient issue to update the routing repo. we could put NOT_CREATED status for that null partition in the map and log error ?

It could be a good idea but let's confirm if that's indeed some transient issue to update the routing repos otherwise the suggested change may hide the real NPE issue